src/users/service/userService.ts

```diff
@@ -53,27 +48,15 @@ export class UsersService {
     return user;
   }
 
-  remove(id: number): Promise<User> {
-    const user =this.usersRepository.remove(id);
+  remove(id: number,uid:string): Promise<User> {
+    const user =this.usersRepository.remove(id,uid);
     if (!user) {
       throw new NotFoundException(`User with ID=${id} not found`);
     }
```

Comment:
Test and attach screenshot, how is it working))))


add await to make this work